soc/arm/atmel_sam0/common/Kconfig.defconfig.series

```diff
@@ -11,6 +11,9 @@ config ADC_SAM0
 config COUNTER_SAM0_TC32
 	default COUNTER
 
+config DAC_SAM0
+	default DAC
```

Comment:
This causes issues when enabling an external dac on a samd51 which has no builtin dac it tries to build the dac_sam0 driver